app/js/TabWrangleOption.js

```diff
@@ -37,12 +39,27 @@ export default class TabWrangleOption extends React.Component<Props> {
               type="radio"
               value={option.name}
             />
+            
             <label className="form-check-label" htmlFor={option.name}>
-              {option.text}
+              {option.text} {" "}
+              <Tooltip
+            html={(
+            <div>
+              <b>{chrome.i18n.getMessage(`options_dedupe_option_${option.name}_label`)}</b>
+            <p>{chrome.i18n.getMessage(`options_dedupe_option_${option.name}_description`)}</p>
+            </div>
+            )}
+            position="top"
+            arrow="true"
+            trigger="mouseenter"
+            inertia="true"
+            size="small"
+            > <i className="fas fa-question-circle" /></Tooltip>
             </label>
+            
           </div>
         ))}
-        <div className="row">
+        {/* <div className="row">
```

Comment:
These descriptions should still be here. Could you add these back in?

Reply:
I moved them to small tooltip icons to make it more compact.
Should I revert these changes?